index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Portfolio: Mobile Skeleton</title>
+        <link rel="stylesheet" href="index.css">
+    </head>
+
+    <body>
+        <main id="main-layout">
+            <header class="flex-item">
+                <nav class="nav-bar">
+                    <div class="nav-bar-item">
+                        <a href="#" title="logo">Welcome</a>
+                    </div>
+                    <div class="nav-bar-item">
+                        <img src="./static/images/hamburger-menu-icon.png" title="Menu Options" alt="Menu" width="30" height="30">
+                    </div>
+                </nav>
+            </header>
+
+            <div class="cont-preview flex-item">
+                <div>
+                    <header class="cont-header">
+                        <span>Hello there! I'm Steven.</span>
+                        <span>Software Developer</span>
+                    </header>
+                    <div class="cont-desc">
+                        I have been a Software Developer for 5 years now, working with leading tech companies in developing
+                        cutting-edge technologies. I'm great at customizing websites as well, creating suitable APIs that swiften
+                        and render testing and debugging easy.
+                        If you're looking to hire a software developer, well, I'm your guy.
+                    </div>
+                    <div class="lets-connect">
+                        <div class="socio-hub">
+                            <img src="./static/images/github-icon.png" title="GitHub" alt="GitHub logo">
+                        </div>
+                        <div class="socio-hub">
+                            <img src="./static/images/linkedin-icon.png" title="Linkedin" alt="Linkedin logo">
+                        </div>
+                        <div class="socio-hub">
+                            <img src="./static/images/2-fingers-icon.png" title="2fingers" alt="2fingers logo">
+                        </div>
+                        <div class="socio-hub">
+                            <img src="./static/images/twitter-icon.png" title="Twitter" alt="Twitter logo">
+                        </div>
+                        <div class="socio-hub">
+                            <img src="./static/images/m-icon.png" title="M" alt="M logo">
```

Comment:
- [ ] Good job so far by implementing the request from the previous reviewer, but there is an issue you still working on. You should use the `ul` tag to group the list of icons and each icon should have a `li` tag with a link (`a` tag).
Example below:
```
<ul>
   <li><a href="your_link_here"><img src="..." alt="logo"></a></li>
   <li><a href="your_link_here"><img src="..." alt="logo"></a></li>
   ...
</ul>
```

Reply:
Fixed